packages/descendants/src/DescendantsReducer.ts

```diff
@@ -38,26 +36,27 @@ export type DescendantsReducerType<T extends HTMLElement> = Reducer<
 >;
 
 /**
+ * A function matching the React.Reducer signature,
+ * that accepts a `DescendantsList` as current state,
+ * and custom `action` object to modify the descendants list.
  *
- * Establishes a state with a `descendants` list, and a `dispatch` function to modify the descendants list
- *
- * @param state {@link DescendantsState} The current reducer state
+ * @param currentState {@link DescendantsState} The current reducer state
  * @param action {@link DescendantsReducerAction} The reducer action type
  * @returns A modified {@link DescendantsState} object
  */
 export const descendantsReducer = <T extends HTMLElement>(
-  state: DescendantsState<T>,
+  currentState: DescendantsState<T>,
```

Comment:
is the fact that we changed the type here going to have any other downstream consequences?

Reply:
(i.e. do we need to update usage in any other component)

Reply:
No, this is internal, and shouldn't be used on it's own
The Provider pulls this property off and uses it